pytket/extensions/qulacs/__init__.py

```diff
@@ -11,7 +11,6 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Module for conversion from tket primitives to Qulacs primitives."""
```

Comment:
why removing these?

Reply:
Coverage checks require there to be an entry for each module (folder/file) in the library; these lines are supposed to be docstrings for that particular module, but they end up getting in the way a lot when rendering the docs webpage. Unless we add proper divisional headings, or rewrite all of these entry lines with the intent to be viewed on the docs site, it is easier to remove them so the docs only contain the actual contents of the library